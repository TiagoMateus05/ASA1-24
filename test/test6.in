15 300
10 5 4 14 5 10 14 1 6 14 13 8 2 6 8
3 9 8 3 5 12 13 14 2 5 4 11 15 9 5
5 15 14 11 13 5 7 5 4 13 14 2 4 13 7
12 8 7 4 1 2 13 12 15 2 11 5 15 6 5
8 6 9 3 9 6 2 2 3 4 11 4 7 5 11
14 12 1 14 15 11 8 6 14 1 2 11 11 12 12
3 6 11 8 13 4 9 5 4 15 2 10 5 7 4
11 15 15 15 4 14 5 9 12 2 7 8 12 13 3
2 1 8 3 2 7 13 9 3 10 14 12 9 1 14
5 2 13 12 3 7 11 7 6 10 5 2 1 7 5
3 8 3 11 6 4 11 4 8 4 8 13 15 6 3
15 4 7 5 3 8 14 3 4 11 13 2 4 15 10
1 10 7 8 12 8 7 3 15 5 6 6 2 12 7
10 6 4 3 15 11 4 7 11 7 4 11 4 9 5
3 10 12 14 4 2 1 10 1 2 8 9 8 4 10
14 2 3 3 6 14 1 5 11 15 12 7 12 5 2 8 10 4 2 5 7 8 7 14 4 7 14 14 12 3 5 7 8 12 1 5 6 6 1 3 11 11 14 1 6 10 1 13 6 2 1 10 4 12 3 9 2 1 10 5 1 10 4 6 2 9 9 2 9 7 1 12 15 2 3 9 5 5 8 6 3 7 14 9 2 1 9 10 4 3 4 12 11 4 2 3 10 9 1 4 1 12 14 15 11 7 14 7 13 10 5 12 8 8 10 2 2 6 6 3 1 8 10 2 13 15 14 14 8 10 8 7 14 4 5 8 10 7 9 8 1 6 5 3 10 5 8 13 2 12 12 10 5 11 10 11 8 4 4 12 11 6 3 13 6 1 11 15 2 1 11 6 8 4 1 10 10 5 11 14 13 9 14 5 8 12 1 12 6 7 4 12 11 12 2 5 4 1 9 1 7 4 6 15 9 15 15 14 15 8 1 4 9 15 6 9 1 5 3 6 13 4 4 6 10 12 6 3 12 11 2 13 4 5 6 8 5 11 6 4 1 1 12 15 5 7 12 8 12 12 6 14 5 13 4 14 2 12 8 5 10 5 8 5 2 6 9 8 14 11 10 8 15 12 5 2 13 10 4 11 2 14 9 10 13 6 6 12 2 5 10 2 3 10 3 11 7 8 3 14
2