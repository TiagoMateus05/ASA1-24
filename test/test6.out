1
(((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((((14 2) 3) 3) 6) 14) 1) 5) 11) 15) 12) 7) 12) 5) 2) 8) 10) 4) 2) 5) 7) 8) 7) 14) 4) 7) 14) 14) 12) 3) 5) 7) 8) 12) 1) 5) 6) 6) 1) 3) 11) 11) 14) 1) 6) 10) 1) 13) 6) 2) 1) 10) 4) 12) 3) 9) 2) 1) 10) 5) 1) 10) 4) 6) 2) 9) 9) 2) 9) 7) 1) 12) 15) 2) 3) 9) 5) 5) 8) 6) 3) 7) 14) 9) 2) 1) 9) 10) 4) 3) 4) 12) 11) 4) 2) 3) 10) 9) 1) 4) 1) 12) 14) 15) 11) 7) 14) 7) 13) 10) 5) 12) 8) 8) 10) 2) 2) 6) 6) 3) 1) 8) 10) 2) 13) 15) 14) 14) 8) 10) 8) 7) 14) 4) 5) 8) 10) 7) 9) 8) 1) 6) 5) 3) 10) 5) 8) 13) 2) 12) 12) 10) 5) 11) 10) 11) 8) 4) 4) 12) 11) 6) 3) 13) 6) 1) 11) 15) 2) 1) 11) 6) 8) 4) 1) 10) 10) 5) 11) 14) 13) 9) 14) 5) 8) 12) 1) 12) 6) 7) 4) 12) 11) 12) 2) 5) 4) 1) 9) 1) 7) 4) 6) 15) 9) 15) 15) 14) 15) 8) 1) 4) 9) 15) 6) 9) 1) 5) 3) 6) 13) 4) 4) 6) 10) 12) 6) 3) 12) 11) 2) 13) 4) 5) 6) 8) 5) 11) 6) 4) 1) 1) 12) 15) 5) 7) 12) 8) 12) 12) 6) 14) 5) 13) 4) 14) 2) 12) 8) 5) 10) 5) 8) 5) 2) 6) 9) 8) 14) 11) 10) 8) 15) 12) 5) 2) 13) (10 4)) 11) ((2 14) 9)) 10) 13) 6) 6) 12) 2) 5) 10) 2) (3 10)) (3 11)) 7) 8) (3 14))
